platform/admin/ui/src/main/webapp/js/views/installer/AnonClaims.view.js

```diff
@@ -81,8 +81,6 @@ define([
                 this.valObj.get("claims").availableClaims = this.valObj.get("claims").availableClaims.sort();
                 this.valObj.get("claims").availableClaims.push("Add Custom Attribute...");
             }
-
-            Backbone.ModelBinder.SetOptions({modelSetOptions: {validate: true}});
```

Comment:
This sets the property globally which we don't want. It was a mistake I made when I added the AnonClaims and it was causing problems with the configuration page.